Content.Shared/_DV/CosmicCult/Components/Examine/CosmicMarkGodComponent.cs

```diff
@@ -0,0 +1,9 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._DV.CosmicCult.Components.Examine;
+
+/// <summary>
+/// Marker component for The Unknown. We also use this to detect its spawn through CultRule!
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class CosmicMarkGodComponent : Component;
```

Comment:
If the `Mark` in here means `Marker`, it's superfluous
```suggestion
public sealed partial class CosmicGodComponent : Component
```